src/city.ts

```diff
@@ -12,13 +12,13 @@ const KEYS = [
 
 let convertedCityList: ICity[] = [];
 // Get a list of all cities.
-function getAllCities(keys: string[]): ICity[] {
+function getAllCities(keys?: string[] = KEYS): ICity[] {
```

Comment:
getting this err while running tests - `typescript Parameter cannot have question mark and initializer.`

Reply:
Ohh my bad